network.go

```diff
@@ -1938,6 +1938,31 @@ func (n *network) TableEventRegister(tableName string, objType driverapi.ObjectT
 	return nil
 }
 
+func (n *network) UpdateIpamConfig(ipV4Data, ipV6Data []driverapi.IPAMData) {
+
+	ipamV4Config := make([]*IpamConf, len(ipV4Data))
+	ipamV6Config := make([]*IpamConf, len(ipV6Data))
+
+	for i, data := range ipV4Data {
+		ic := &IpamConf{}
+		ic.PreferredPool = data.Pool.String()
+		ic.Gateway = data.Gateway.IP.String()
+		ipamV4Config[i] = ic
+	}
+
+	for i, data := range ipV6Data {
```

Comment:
same here...we are not sending any IPV6 data.  why do we need it here ?

Reply:
Same as above.